src/inbox.erl

```diff
@@ -92,16 +98,33 @@ dispatcher_routine() ->
 %% Internal functions
 %%%---------------------------
 
+-spec base_post(Vertex :: binary(), Event :: event(), Notification :: notification()) -> ok.
+base_post(Vertex, Event, Notification) ->
```

Comment:
Konwencja wskazywała by na nazwanie tej funkcji raczej `do_post`

Reply:
Czyja konwencja? Choć przyznaję, w tym przypadku ma to sens